app/src/main/java/com/fastaccess/ui/modules/repos/issues/create/CreateIssueActivity.java

```diff
@@ -213,6 +215,16 @@ public static void startForResult(@NonNull Activity activity, @NonNull Intent in
             }
         }
         if (isFeedback) setTitle(R.string.submit_feedback);
+        if (BuildConfig.DEBUG && isFeedback) {
+            new AlertDialog.Builder(this)
+                    .setTitle("You are currently using a debug build")
+                    .setMessage("If you have found a bug, please report it on slack." + "\n" +
+                            "Feature requests can be submitted here." + "\n" + "Happy Testing")
+                    .setPositiveButton(android.R.string.ok, (dialogInterface, i) -> {
```

Comment:
what I meant is that it could be simplified to 
```
new AlertDialog.Builder(this)
                    .setTitle("You are currently using a debug build")
                    .setMessage("If you have found a bug, please report it on slack." + "\n" +
                            "Feature requests can be submitted here." + "\n" + "Happy Testing")
                    .setPositiveButton(android.R.string.ok, null)
                    .show();
```